test/controllers/sicAndCompliance/ComplianceIntroductionControllerSpec.scala

```diff
@@ -99,9 +94,21 @@ class ComplianceIntroductionControllerSpec extends VatRegSpec {
       callAuthorised(ComplianceIntroductionController.submit, mockAuthConnector) {
         result =>
           status(result) mustBe SEE_OTHER
-          redirectLocation(result) mustBe Some( //TODO below should be routing to a different controller, once we have that controller
-            controllers.sicAndCompliance.labour.routes.CompanyProvideWorkersController.show().url
-          )
+          result redirectsTo s"$contextRoot/compliance/provide-workers"
+      }
+    }
+  }
+
+  s"POST ${sicAndCompliance.routes.ComplianceIntroductionController.submit()} with financial SIC code selection" should {
+
+    "redirect the user to the first question about financial compliance" in {
+      when(mockS4LService.fetchAndGet[SicStub]()(any(), any(), any())).thenReturn(Future.successful(
+        Some(SicStub(Some("70221123"), Some("64921123"), None, None))
+      ))
+      callAuthorised(ComplianceIntroductionController.submit, mockAuthConnector) {
+        result =>
+          status(result) mustBe SEE_OTHER
```

Comment:
here we don't need to assert the SEE_OTHER status, it's included in the redirectsTo i think